dll/win32/shell32/wine/classes.c

```diff
@@ -334,6 +334,108 @@ BOOL HCR_GetIconA(LPCSTR szClass, LPSTR szDest, LPCSTR szName, DWORD len, int* p
 	return ret;
 }
 
+#ifdef __REACTOS__
+BOOL HCU_GetIconW(LPCWSTR szClass, LPWSTR szDest, LPCWSTR szName, DWORD len, int* picon_idx)
+{
+	HKEY	hkey;
+	WCHAR	sTemp[MAX_PATH];
+	BOOL	ret = FALSE;
+
+	TRACE("%s\n",debugstr_w(szClass) );
+
+	swprintf(sTemp, L"%s\\DefaultIcon", szClass);
+
+	if (!RegOpenKeyExW(HKEY_CURRENT_USER, sTemp, 0, KEY_READ, &hkey))
+	{
+	  ret = HCR_RegGetIconW(hkey, szDest, szName, len, picon_idx);
+	  RegCloseKey(hkey);
+	}
+
+        if(ret)
+            TRACE("-- %s %i\n", debugstr_w(szDest), *picon_idx);
+        else
+            TRACE("-- not found\n");
```

Comment:
Indentation error?

Reply:
Yep, indeed. The tabs are used instead of spaces in the whole code.

Reply:
No wrong. You wrote a new function here entirely. So 4spaces should be used for indentation within the whole function, instead of any tabs (or even worse space-tab-mixture as it is partially done within this function, e.g. at line current line 352).
Contrary the line 357 is the only proper one currently because it uses "AmountOfSpaces modulo 4 == 0" and no tabs at all.

Reply:
Ah, then I probably should use the spaces in all newly added functions...

Reply:
Done. Fixed indentation for all new code. Also improved ANSI to Unicode conversion a bit. Thanks. :slightly_smiling_face: